### DISCUSSION TITLE
YouTube Revanced can't select the right audiotrack

### DISCUSSION THREAD

**Discussion by Gxinz** [ReVanced/revanced-discussions#1104] [Question] [answered]
So I'm in Germany, when I watch german YouTubers I can't select the german audiotrack. It only let me select the English one. And all of a sudden, the audio track option is completely gone whether I turn it on or off in the float menu...
![Screenshot_20241129-032735](https://github.com/user-attachments/assets/9e5a9ca7-2580-4080-8d0f-e8e18dd676f9)
![Screenshot_20241129-033741](https://github.com/user-attachments/assets/99e78138-f1e8-4491-b797-c1dbe699c637)

**Comment by oSumAtrIX** [comment, marked as answer]
Use the latest patches. If the spoof video stream patch falls back to the iOS client, audiotracks will not be available.

**Reply by Gxinz** [reply]
I did use the latest patch and suggested YT version.
Edit: Got it, ReVanced -> Miscellaneous -> Spoof video streams -> Client : iOS not Android
